### ISSUE TITLE
Bug when selecting mails by size

### ISSUE THREAD

**Issue by outini** [outini/pymailq#13]
We actually can specify minimum size without maximum size.

```
PyMailq (sel:1313)> select size +1000
*** Syntax error: minimum size is greater than maximum size
PyMailq (sel:1313)> select size -0 +1000  
*** Syntax error: minimum size is greater than maximum size
PyMailq (sel:1313)> select size +1000 -100000
PyMailq (sel:1185)> 
```

**Comment by outini** [comment]
Fixed in version 0.7.0

**Comment by outini** [comment]
This bug is not fixed.
Unittests has to be improved.
